common/beerocks/bcl/source/beerocks_socket_thread.cpp

```diff
@@ -222,9 +229,9 @@ bool socket_thread::verify_cmdu(message::sUdsHeader *uds_header)
                 break;
             }
 
-            //} else {
-            //    THREAD_LOG(WARNING) << "Not Intels VS message!";
-            //}
+            } else {
+                THREAD_LOG(WARNING) << "Not Intels VS message!";
```

Comment:
Typo change to `Not an Intel vendor specific message `

Reply:
Will be fixed.
I am also changing it to an INFO message (maybe it should even be DEBUG?) - I don't think we should throw warning logs for messages of other vendors, just ignore them.

Reply:
I think that for now it should be warning because we don't expect this. In the future we can change that